go.mod

```diff
@@ -31,15 +31,15 @@ require (
 	github.com/stretchr/testify v1.8.4
 	github.com/tidwall/gjson v1.17.0
 	github.com/vmihailenco/msgpack/v5 v5.4.1
-	go.opentelemetry.io/otel v1.21.0
-	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp v0.44.0
-	go.opentelemetry.io/otel/metric v1.21.0
-	go.opentelemetry.io/otel/sdk v1.21.0
-	go.opentelemetry.io/otel/sdk/metric v1.21.0
-	go.opentelemetry.io/proto/otlp v1.0.0
+	go.opentelemetry.io/otel v1.23.1
+	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp v1.23.1
```

Comment:
This is the only change that gave me pause, but here's why I think it's OK:

- [dependabot took otlpmetrichttp](https://github.com/honeycombio/refinery/pull/983) to [0.45.0](https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.22.0) which is the version number for experimental components released along with the rest of stable OTel Go v1.22.0
- `otlpmetrichttp` then had a stable release candidate in [v1.23.0-rc1](https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.23.0-rc.1)
- `otlpmetrichttp` went stable in [v1.23.0](https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.23.0) and then was not touched in a patch to `sdk/metric` in [v1.23.1](https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.23.1)